docs/DeveloperGuide.md

```diff
@@ -105,30 +108,75 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-<u>**Use case: Delete a candidate**</u>
+<u>**Use case: UC03 - Delete a candidate**</u>
 
 **MSS**
 
-1. User requests to list candidates.
-2. HR Manager shows a list of candidates.
-3. User requests to delete a specific candidate.
-4. HR Manager deletes the corresponding candidate and displays result.
+1. User requests to <u>list all candidates (UC02)</u>.
+2. User requests to delete a specific candidate.
+3. HR Manager deletes the corresponding candidate and displays result.
 
    Use case ends
 
 **Extensions**
 
-* 2a. The list of candidates is empty.
+* 3a. The given index is invalid.
+
+    * 3a1. HR Manager shows an error message.
+
+      Use case resumes at step 2.
+
+<u>**Use case: UC04 - Add a position**</u>
+
+**MSS**
+
+1. User requests to add a new position, with the title of the new job position.
+2. User can see the added job position.
 
-  Use case ends.
+   Use case ends.
+
+**Extensions**
+
+* 1a. The format when adding a position is incorrect.
+
+    * 1a1. HR Manager shows an error message.
+
+      Use case ends.
```

Comment:
Can also add the extension when the position already exists in the HR Manager. We can also do the same for add_c